submissions/html/assignment1/2203031080131/q5.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+
+    <h1>Survey Form</h1>
+    <form >
+        
+        
+        <label for="name">Full Name:</label>
+        <input type="text" id="name" name="name" >
+        <br><br>
+
+        
+        <label for="male">Male</label>
+        <input type="radio" id="male" name="gender" value="male" >
+        <label for="female">Female</label>
+        <input type="radio" id="female" name="gender" value="female" >
+        <br><br>
+
+        <label for="C">C language</label>
+        <input type="checkbox" id="C" name="interests" value="C">
+        <label for="java">Java</label>

```

Comment:
For Radio Box, Checkbox, put the input element first and then the label for a better UX!